content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -107,43 +129,48 @@ pub struct Context<'a, 'b, 'c, 'info, T> {
     /// Bump seeds found during constraint validation. This is provided as a
     /// convenience so that handlers don't have to recalculate bump seeds or
     /// pass them in as arguments.
-    pub bumps: BTreeMap<String, u8>,
+    /// Type is the bumps struct generated by #[derive(Accounts)]
+    pub bumps: T::Bumps,
 }
 ```
 
 `Context` is a generic type where `T` defines the list of accounts an
 instruction requires. When you use `Context`, you specify the concrete type of
 `T` as a struct that adopts the `Accounts` trait (e.g.
-`Context<AddMovieReviewAccounts>`). Through this context argument the
+`Context<AddMovieReviewAccounts>`). Through this `context` argument the
 instruction can then access:
```

Comment:
I think we can delete most of this section. It duplicates the descriptions of each Context item we have in the next section.

Replace it with:

> The first argument of every instruction handler must be `Context`. Context takes a generic of your Accounts struct, eg,  if `AddMovieReview` was the struct holding the accounts, the context for the add_movie_review() function would be `Context<AddMovieReview>`,

Then add a ContentBox (see CONTRIBUTING) with:

> Yes, the Accounts struct is typically named the same thing as the instruction handler, just in TitleCase. Eg, the struct with the accounts for add_movie_review() is called AddMovieReview! 


BTW I should have asked you to do the same in `intro-to-anchor` during #349, since that chapter does the same thing - pastes the Rust code describing the struct, then describes the struct straight afterwards!|

Reply:
Couldn't find ContentBox in the CONTRIBUTING. Do you mean callout by any chance?

Reply:
Callout! Yes.

Reply:
done